packages/core/src/theme.d.ts

```diff
@@ -0,0 +1,143 @@
+import * as CSS from 'csstype'
+import { SystemStyleObject } from '@styled-system/css'
+import { Theme as StyledSystemTheme } from 'styled-system'
```

Comment:
I'm guessing the plan is to eventually have the entire Theme interface defined in this repo, right?